src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinatorModel.swift

```diff
@@ -410,4 +414,73 @@ class ExposureSubmissionCoordinatorModel {
 			fatalError("Cannot set submission consent, no corona test type is set")
 		}
 	}
+
+	/// Check SRS Flow Prerequisites.
+	/// - Parameters:
+	/// 	- isLoading: The callback that should be execute while fetching the self service parameters from app configuration.
+	/// 	- completion: The completion handler, that has given the regarding `SRSPreconditionError`, in case of a precondition fault.
```

Comment:
```suggestion
	/// 	- completion: The completion handler, that provides the corresponding `SRSPreconditionError` in case of a precondition fault.
```

Reply:
fixed